Apps/W1/HybridGP/app/src/Migration/Support/PurchaseOrders/GPPOMigrator.codeunit.al

```diff
@@ -6,140 +6,166 @@ codeunit 40108 "GP PO Migrator"
 
     procedure MigratePOStagingData()
     var
-        GPPOPPOHeader: Record "GP POPPOHeader";
+        GPPOP10100: Record "GP POP10100";
         PurchasesPayablesSetup: Record "Purchases & Payables Setup";
         CompanyInformation: Record "Company Information";
         PurchaseHeader: Record "Purchase Header";
+        GeneralLedgerSetup: Record "General Ledger Setup";
+        CurrencyExchangeRate: Record "Currency Exchange Rate";
+        HelperFunctions: Codeunit "Helper Functions";
         PurchaseDocumentType: Enum "Purchase Document Type";
         PurchaseDocumentStatus: Enum "Purchase Document Status";
         CountryCode: Code[10];
+        CurrencyCode: Code[10];
     begin
-        if not GPPOPPOHeader.FindSet() then
+        GPPOP10100.SetRange(POTYPE, GPPOP10100.POTYPE::Standard);
+        GPPOP10100.SetFilter(POSTATUS, '%1..%2', 1, 4);
```

Comment:
Unfortunatelly 1%..%2 needs to be moved to the locked label. You can also do SetRange(POSTATUS,1,4) it is the same syntax.